### DISCUSSION TITLE
Any easier way not to overwrite the text-color for select on navbar?

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1380] [Q&A] [answered]
Hi

I have a navbar with blue back-ground and text-white. I also have included a select to it. The select inherits the text-color of the nav, perhaps expected, is there an easier way to ensure that the select does not inherit the text color of the parent navbar?

I know I can just add a class of text-black on the select to make it remain default.

![image](https://user-images.githubusercontent.com/5799306/203389354-1ad7829b-4481-4c1a-afc8-ddacefcc88d6.png)

https://play.tailwindcss.com/MYLvSP1VDo

Thanks,

**Comment by saadeghi** [comment, marked as answer]
As you said, text color will be inherited unless you add your own text color to the select.  
However, I think using `select select-ghost` fits better for here. it makes the select background transparent so both background color and text color will be inherited from your custom colors.  
It also looks better: https://play.tailwindcss.com/26sWokDGRH

Let me know if you have any questions.

**Reply by Mashiane** [reply]
Thank you so much for the good pointers, appreciated!
